data/src/main/java/org/akvo/flow/data/net/FlowApiService.java

```diff
@@ -20,6 +20,16 @@
 
 package org.akvo.flow.data.net;
 
-public interface FlowApiService {
+import org.akvo.flow.data.entity.ApiLocaleResult;
 
+import retrofit2.http.GET;
+import retrofit2.http.Headers;
+import retrofit2.http.Url;
+import rx.Observable;
+
+interface FlowApiService {
+
+    @GET
```

Comment:
If possible it would be great as it also takes care of the encoding so less code written.